DEVBOX.md

```diff
@@ -0,0 +1,60 @@
+# Devbox Setup Guide
+
+## Prerequisites
+- macOS (Darwin) or Linux
+- Nix package manager
+
+## Installation
+
+### 1. Install Devbox
+```bash
+curl -fsSL https://get.jetpack.io/devbox | bash
+```
+
+### 2. Initialize Devbox Environment
+```bash
+cd /Users/raj.surve/DKP/charts
+devbox shell
+```
+
+### 3. Verify Tools
+```bash
+helm version
+ct version
+yamllint --version
+yamale --version
+kubectl version --client
+kind version
+```
+
+## Usage
+
+### Start Development Shell
+```bash
+devbox shell
```

Comment:
in future PRs, please consider adding `direnv` to automatically drop into devbox shell.